cf-servicesinfo.rb

```diff
@@ -163,23 +136,25 @@ def get_space_service_instances(space_guid, service_plans)
 stamp = Time.now.to_i
 
 CSV.open("cf-orgs-services-managers-#{stamp}.csv", "wb") do |csv|
-  csv << ["Org","Org Managers","Space", "Space Developers", "Space Developers", "Space GUID"] + get_service_plan_headers(@service_plans)
+  csv << ["Org","Org Managers","Space", "Space Developers", "Space Developers"] + get_service_plan_headers(@service_plans)
```

Comment:
this should be "Org","Org Managers","Space", **"Space Managers"**, "Space Developers"